tools/wt_verify/wt_verify.py

```diff
@@ -588,10 +595,9 @@ def main():
 
     if "dump_pages" in command:
         parsed_data = parse_dump_pages()
-    elif "dump_blocks" in command:
-        parsed_data = parse_dump_blocks()
     else:
-        assert False, f"Unexpected command '{command}'"
+        # If it's "dump_blocks" then do parse_dump_blocks()
+        parsed_data = parse_dump_blocks()
```

Comment:
I think the last refactor left this in a bad state - there's no longer any checking that `command` contains something sane, as we'll always fall into the `else` case.

Reply:
The parser.add_argument has included a checking `choices=['dump_blocks','dump_pages']` for the command, I think one validation is sufficient as the input is in a single component. What is your opinion? @etienneptl

Reply:
I agree with you, this should be enough. Should we add a comment saying the argument parser already checks the possible values for future readers?

Reply:
@korteland, would a comment be enough?

Reply:
Sorry - I had a few other things on today and didn't see this thread. I didn't realise the parser already checked this for us, in that case I'm happy with the change as-is :-)